src/main/resources/assets/paths/datasetById.yaml

```diff
@@ -1,3 +1,56 @@
+patch:
+  summary: Patch Dataset
+  description: |
+    This API allows Admins, Chairpersons, and Data Submitters to patch a limited set of dataset
+    properties. Data Location must conform to the schema defined in
```

Comment:
the properties and the dataset name too right?

Reply:
Yes, thank you, I'll add that to the description.